src/java/org/apache/cassandra/db/compaction/AbstractCompactionTask.java

```diff
@@ -151,7 +152,12 @@ public int execute()
         }
     }
 
-    public Throwable cleanup(Throwable err)
+    public Throwable rejected(Throwable t)
+    {
+        return cleanup(t);
+    }
+
+    protected Throwable cleanup(Throwable err)
```

Comment:
can we make this public again ?
it is used in CNDB: https://github.com/riptano/cndb/blob/ebf8bbb77b8b1be7a13b9427b88e6c37be23dc47/cndb-tasks/src/main/java/com/datastax/cndb/compactor/CompactionTasksOrganizer.java#L952

Reply:
Why not call `rejected`?

Reply:
I have updated the code to use `rejected`, thanks